Java/sort/BubbleSort.java

```diff
@@ -3,12 +3,14 @@ static void bubbleSort(int[] arr) {
         int n = arr.length;  
         int temp = 0;  
          for(int i=0; i < n; i++){  
+            boolean sorted = true;
              for(int j=1; j < (n-i); j++){
                  if(arr[j-1] > arr[j]){
                      //swap elements
                      temp = arr[j-1];
                      arr[j-1] = arr[j];
                      arr[j] = temp;
+                    sorted = false;
                  }
                           
```

Comment:
@yogitasaini then in that case, shouldn't there be a check here in this line that checks for the value of `sorted`? If it's true, then there is no need to execute the loop anymore, so the loop should stop? I think you should add a check here.